lecli/apiutils.py

```diff
@@ -1,26 +1,33 @@
 import ConfigParser
 import base64
-import datetime
 import hashlib
 import hmac
 
+import datetime
+from appdirs import user_config_dir
+
+CONFIG = ConfigParser.ConfigParser()
 
-configfile = 'config.ini'
-config = ConfigParser.ConfigParser()
 
 def load_config():
-  files_read = config.read(configfile)
-  if len(files_read) != 1:
-    print "Error: Config file '%s' not found" % configfile
-    exit(1)
-  if not config.has_section('Auth'):
-    print "Error: Config file '%s' is missing Auth section" % configfile
-    exit(1)
+    config_file = user_config_dir('lecli') + '/' + 'config.ini'
```

Comment:
os.path.join(user_config_dir('lecli'), 'config.ini')
to make it platform independant